requirements.sh

```diff
@@ -31,4 +31,5 @@ sudo apt install -y cmake g++ make build-essential autoconf automake uuid-dev
 sudo apt install -y libtool libboost-dev libboost-system-dev libboost-thread-dev libpq-dev libssl-dev libz-dev
 sudo apt install -y python-dbus python-dev python3-dev python3-pip
 sudo apt install -y sqlite3 libsqlite3-dev
+sudo apt install -y pkg-config
```

Comment:
Please open a corresponding pull request for https://github.com/foglamp/foglamp-pkg `control` file `Depends:`  field.

Reply:
"pkg-config" is required only at compile time by cmake to find other packages, it is not required at run-time. So probably it is not required to add pkg-config to control fie "Depends" field ?

Reply:
Yeah. It should be fine then.